src/trajectory_tracking.c

```diff
@@ -0,0 +1,67 @@
+
+#include <stdlib.h>
+#include <math.h>
+
+#include "trajectory_tracking.h"
+
+
+/* public module prototypes */
+uint8_t tracy_linear_controller(const tracking_error_t * current_error,
+                                const reference_velocity_t * reference_velocity,
+                                feedback_rule_t * output);
+
+uint8_t tracy_set_controller_params(float fancy_greek_c, float g);
+
+/* module variables */
+
+static float param_fancy_greek_c = DEFAULT_PARAM_FANCY_GREEK_C;
```

Comment:
The letter is Zeta ;) But I think "damping coefficient" would be more descriptive.